website/docs/faqs/testing-sources.md

```diff
@@ -22,3 +22,11 @@ $ dbt test --select source:jaffle_shop.orders
 ```
 
 Yep, we know this syntax is a little less than ideal, so we're hoping to improve it in a future release. Check out the [model selection syntax](node-selection/syntax) for more examples!
+
+
+:::info Node selection syntax 
+In dbt 0.21.0, the node selection syntax was standardised (https://github.com/dbt-labs/dbt-core/pull/3791) to use `--select` everywhere. Before this, some commands like `dbt run` and `dbt test` used `--models` instead.  
+ 
+Older versions show the error because argparse (https://docs.python.org/3/library/argparse.html#allow-abbrev) is expanding `--select` to `--selector`, a different flag.
```

Comment:
```suggestion
Older versions still show an error because [argparse](https://docs.python.org/3/library/argparse.html#allow-abbrev) is expanding `--select` to `--selector`, which is a different flag.
```